pkg/kdk/config.go

```diff
@@ -371,8 +372,27 @@ func (c *KdkEnvConfig) Exec(command string) error {
 	commandString := fmt.Sprintf("%s %s", c.SSHCommandString(), command)
 	log.Infof("executing ssh command: %s", commandString)
 	commandMap := strings.Split(commandString, " ")
```

Comment:
NIT: It's not really a map.  More like a list

Reply:
That part wasn't my code. Just moved it around for DRY.

Reply:
You're right though. It's a slice `[]string`